### PR TITLE
Create Demo App

### PR DESCRIPTION
**Pull request by jansule**
Create basic Demo App. Added geostyler logo as favicon.

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "geostyler-demo",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@types/jest": "23.3.9",
+    "@types/node": "10.12.2",
+    "@types/react": "16.4.18",
+    "@types/react-dom": "16.0.9",
+    "antd": "^3.10.4",
```
Comment (KaiVolland) [inline package.json]: Please remove all carets

File: src/logo.svg [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 841.9 595.3">
```
Comment (KaiVolland) [inline src/logo.svg]: Is this the GeoStyler Logo? Or the CRA Logo? If CRA please remove.

File: src/react-app-env.d.ts [hunk 3]
```diff
@@ -0,0 +1,3 @@
+/// <reference types="react-scripts" />
+declare module 'geostyler';
+declare module 'geostyler/dist/Component/Style/Style';
```
Comment (KaiVolland) [inline src/react-app-env.d.ts]: Shouldn't the geostyler package declare these modules?

File: src/serviceWorker.ts [hunk 4]
```diff
@@ -0,0 +1,143 @@
+// This optional code is used to register a service worker.
```
Comment (KaiVolland) [inline src/serviceWorker.ts]: I don't think we need this file. Please remove.

File: src/logo.svg (hunk 2 above)
Comment (jansule) [reply src/logo.svg]: It's the GeoStyler Logo.

image

File: src/react-app-env.d.ts (hunk 3 above)
Comment (jansule) [reply src/react-app-env.d.ts]: Yes, it should. But currently it's not doing it. Had to define them to make it running.

**Pull request merged**